examples/src/examples/graphics/lights-baked-ao.tsx

```diff
@@ -1,114 +1,152 @@
+import React from 'react';
 import * as pc from 'playcanvas/build/playcanvas.js';
 import Example from '../../app/example';
+import { AssetLoader } from '../../app/helpers/loader';
+// @ts-ignore: library file import
+import Panel from '@playcanvas/pcui/Panel/component';
+// @ts-ignore: library file import
+import SliderInput from '@playcanvas/pcui/SliderInput/component';
+// @ts-ignore: library file import
+import LabelGroup from '@playcanvas/pcui/LabelGroup/component';
+// @ts-ignore: library file import
+import Label from '@playcanvas/pcui/Label/component';
+// @ts-ignore: library file import
+import BooleanInput from '@playcanvas/pcui/BooleanInput/component';
+// @ts-ignore: library file import
+import BindingTwoWay from '@playcanvas/pcui/BindingTwoWay';
```

Comment:
I'm confused why this isn't:

```
import { BindingTwoWay, BooleanInput, Label, LabelGroup, Panel, SliderInput } from '@playcanvas/pcui';
```

Reply:
not sure, the old way which is what you suggested wasn't working anymore, so I swapped it to a form from other examples. Perhaps related to tree-shaking changes @vkalpias was doing in pcui?

Reply:
Hoping this is a temporary thing since I've never seen module imports having to be done like this for any other framework.